inst/components/fill.scss

```diff
@@ -22,8 +22,18 @@
   width: 100%;
   height: 100%;
   margin: 0;
-  padding: 1rem;
-  gap: 1rem;
+  padding: $spacer;
+  gap: $spacer;
+}
+
+.bslib-page-title {
+  background-color: $bslib-page-title-bg;
+  color: $bslib-page-title-color;
+  font-size: $h4-font-size;
+  font-weight: 300;
+  padding: $spacer * 0.5;
```

Comment:
I like this with more spacing and I think using `$spacer` gets closer to the `page_navbar()` sizing.

```suggestion
  padding: $spacer;
```


![image](https://github.com/rstudio/bslib/assets/5420529/47173728-5d8b-4122-9f67-b200de4712a7)